README.md

```diff
@@ -282,6 +282,8 @@ When using Input plugins, please be aware that input values get cached between s
       cache_inputs: false
 ```
 
+ Default for backend cache is enabled
```

Comment:
Please give an example of setting the value. Imagine you are a user, wishing to disable the backend cache. What is the exact name of the setting? Where in the configuration tree does the setting go? Thanks!